packages/user-interface/src/tests/mock/data/OverviewPageRequests.mock.tsx

```diff
@@ -0,0 +1,3 @@
+import { getTaken, getZaken } from "./AllRequests.mock";
+
+export const mockRequest = [getZaken, getTaken];
```

Comment:
Kunnen deze niet direct aan de AllRequests geïmporteerd worden als ze nodig zijn?

Reply:
@Borai Ik dacht gewoon voor alle pagina's te definiëren welke calls gedaan worden. Kan je hier ook verschillende versies definiëren bijvoorbeeld (eentje met een error bijvoorbeeld). Geen goed idee?

Reply:
@FrederikCampagne ik zou dat gewoon in de page test zelf doen dan. Anders is deze file alleen maar een import/export bestandje.